Storage/Rust/RustRemoteTabs.swift

```diff
@@ -157,6 +157,45 @@ public class RustRemoteTabs {
 
         return deferred
     }
+
+    public func getClient(fxaDeviceId: String) -> Deferred<Maybe<RemoteClient?>> {
```

Comment:
Note: This function will be used in the `FxAPushHandler.handle` function and needs to use `Deferred` to avoid a refactor of that function. The function can be refactored when the Nimbus rollout code is removed.